src/device/device_ledger.cpp

```diff
@@ -751,7 +790,7 @@ namespace hw {
         this->buffer_send[4] = 0x00;
         offset = 5;
         //options
-        this->buffer_send[offset] = options;
+        this->buffer_send[offset] = 0;
         offset += 1;
         //sec
         memmove(this->buffer_send+offset, sec.data, 32);
```

Comment:
Out of curiosity: isn't this line pointless because the device already knows the view secret key? Giving only the subaddress index seems sufficient to me.

Reply:
ok